templates/new/config/prod.exs

```diff
@@ -1 +1,7 @@
 use Mix.Config
+
+# Configure shoehorn to use the default handler in prod.
+# The default handler will call `:erlang.halt()` when an 
+# OTP application exits.
```

Comment:
I assume that no production Nerves device will want to use `Shoehorn.Handler.Default`, right? That bricks the device if there's not enough time to send an update down before `:erlang.halt()` is called.

Reply:
If this were true, then the over all default should be set to Ignore. and all of this exercise is moot. We would go back to what we had which is that existing projects that suffer an OTP application failure would be zombies.

Reply:
The default handler makes it act as if shoehorn was not there--if an app dies them beam dies.  I don't think it bricks it as much as restarts the whole device

Reply:
I just had this happen to me. An application died shortly after it started and the device went into an infinite reboot. I couldn't firmware update it and the device didn't work so that's why I called it a brick.